src/PackageSubmitter.php

```diff
@@ -46,7 +46,9 @@ private function getPackages(OutputInterface $output): array
     private function getVendorState(array $packages): array
     {
         return [
-            'id'        => $this->hasher->generateInstallIdHash(getcwd()),
+            'id'        => $this->hasher->generateInstallIdHash(
+                $this->packageResolverStrategy->getRootDirectory()
```

Comment:
Sorry, I'm not currently behind a computer, but shouldn't this then be `resolveRootPath` rather than `getRootDirectory`? I do not see the latter defined anywhere.

Reply:
Yes, it should. Thanks for catching this. I had updated this to be correct in `.../magento/vendor/sansec/...`, but clearly didn't copy it out to where I committed from. I had initially created this as `getRootDirectory()` everywhere, but decided it'd be best to keep with the existing naming convention of 'resolveThing()'.